### PR TITLE
Adding halstack tooltip documentation

### PR DESCRIPTION
**Pull request by Mil4n0r**
**Checklist**
_(Check off all the items before submitting)_

- [ ] Build process is done without errors. All tests pass in the `/lib` directory.
- [ ] Self-reviewed the code before submitting.
- [ ] Meets accessibility standards.
- [ ] Added/updated documentation to `/website` as needed.
- [ ] Added/updated tests as needed.

**Description**
Custom tooltip for halstack was already implemented but doc pages were missing.

**Related to  #1964**

### CODE REVIEW

**Comment by Mil4n0r** [comment]
It is required to export DxcTooltip from the library first

**Comment by Mil4n0r** [comment]
For now, halstack-react cannot be upgraded to next version due to using NextJS. We will have to wait for the mono-repo update before we can add the examples.

File: website/screens/components/tooltip/code/TooltipCodePage.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import { DxcFlex, DxcTable } from "@dxc-technology/halstack-react";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import Example from "@/common/example/Example";
+// import basicUsage from "./examples/basicUsage";
+import StatusBadge from "@/common/StatusBadge";
+import TableCode from "@/common/TableCode";
+type Props = {
```
Comment (GomezIvann) [inline website/screens/components/tooltip/code/TooltipCodePage.tsx:10]: Why are you defining a Props type?

Comment (Mil4n0r) [reply website/screens/components/tooltip/code/TooltipCodePage.tsx]: That was unintentional, thank you.

**Review by GomezIvann — DISMISSED** [review]

**Review by GomezIvann — APPROVED** [review]

**Pull request merged**